### ISSUE TITLE
Hiding certain custom commands

### ISSUE THREAD

**Issue by mike-potter** [docksal/docksal#1754]
When adding custom commands to Docksal, we have some common "helper" functions we want to share between commands.  For various reasons (the way we package projects) we'd like those helper functions defined in a script file in the same folder as the commands that use them.

Thus, we end up with a `.helper` script in the `.docksa/commands` folder, and this shows up in the `fin help` output as another custom command.

I'd like to request a feature where "dot files" (or some other pattern you think is better) are not shown in `fin help`. This would allow some custom commands to be hidden and not pollute the help listing.

**Comment by shelane** [comment]
I have helper functions also. I put them in the .docksal folder, but not in the commands folder and then reference them like:
```
source "${PROJECT_ROOT}/.docksal/includes/helpers.sh"
```

**Comment by mike-potter** [comment]
Yep.  In our case the `.docksal/commands` is actually symlinked to another location so we really need them in the same folder.

**Comment by mike-potter** [comment]
If worried about backward compatibility with existing projects that have their commands starting with a dot for some reason, it would be fine if it just excluded subfolders that start with a dot.  After all, dot files/folders are supposed to be "hidden" so seems like it would make sense to not show them as commands in the `fin help`.
